src/database/migrations/20201221203940-create-trip-request.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('TripRequests', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      tripType: {
+        type: Sequelize.STRING
+      },
+      departureDate: {
+        type: Sequelize.DATEONLY,
+      },
+      returnDate: {
+        type: Sequelize.DATEONLY,
+      },
+      reason: {
+        type: Sequelize.STRING
+      },
+      userId: {
+        type: Sequelize.INTEGER,
+        references: { model: 'Users', key: 'id' }
+      },
+      status: {
+        type: Sequelize.STRING,
+        defaultValue: "pending"
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
```

Comment:
can we add is isApproved field?

Reply:
> can we add is isApproved field?

That would be a repetition because `status ` field may have values like `pending, approved or denied`.